### ISSUE TITLE
Is there a way to have multiple games running at the same time and swap between them?

### ISSUE THREAD

**Issue by KateHanami** [ValveSoftware/openvr#1734]
Assuming I can run multiple VR games in a single GPU, ir maybe I could suspend the game that's not in focus

Is there a possible way to run 2 or mire games running concurrently? And the HMD and controllers will display / track the game I'm currently focusing on until I swap them?

For example, the ne Resident Evil 4 remakes is getting a VR mid soon, and I would like to Steam the classic and the remake and swap between them in real time to see the difference 

Is such a thing feasible?

**Comment by Rectus** [comment]
SteamVR does not have multitasking support for regular scene applications. The only way to run multiple applications simultaneously is if at least all but one are overlay applications, and even then it's not supported very well.

**Comment by vrbober** [comment]
Did anything change for the past year? Also isn't steam uses openxr now?

**Comment by Rectus** [comment]
Nope, no multitasking support yet. And SteamVR has supported OpenXR since 2020/2021.
